src/content/community/particle-101-workshop/device-cloud.md

```diff
@@ -22,7 +22,7 @@ In this session, we'll take a look at the Particle CLI and Device Cloud API. If
 ## The Particle CLI
 1.  If you haven't already, [install the Particle CLI](/guide/tools-and-features/cli/photon/). Open a terminal window and type the following command:
 ```bash
-bash <( curl -sL https://particle.io/install-cli )
+bash <( curl -sL https://binaries.particle.io/particle-cli/installer/install-cli)
```

Comment:
should be better to keep it as is and change it over here?  👉  https://github.com/particle-iot/website/blob/master/static/_redirects#L215

Reply:
Sure, I can switch all of those back, it would make it easier to type.

Reply:
Done